src/main/java/opgg/weba/JamPick/domain/IndieApp.java

```diff
@@ -24,11 +24,8 @@
     @Column(name = "is_free")
     private Boolean isFree;
 
-    @Column(name = "detailed_description")
-    private String detailedDescription;
-
-    @Column(name = "release_date")
-    private String releaseDate;
+    @Column(name = "short_description")
+    private String shortDescription;
```

Comment:
I think this field should be in `IndieAppDetail`. Because, `short_description` is affected by langauge.

Reply:
I agree. I'll correct this.